include/cctz/civil_time_detail.h

```diff
@@ -91,6 +91,15 @@ CONSTEXPR_F int days_per_4years(int yi) noexcept {
 CONSTEXPR_F int days_per_year(year_t y, month_t m) noexcept {
   return is_leap_year(y + (m > 2)) ? 366 : 365;
 }
+// The compiler cannot optimize away the check if we use
+// fsanitize=array-bounds.
+// When this function is inlined, m is guaranteed to be in [1, 13).
```

Comment:
Above we say `using month_t = std::int_fast8_t;   // [1:12]`, so we might as well use the same syntax, _"[1:12]"_, for consistency's sake.

Reply:
done